docs/src/guide/installation.md

```diff
@@ -30,6 +30,40 @@ additional build system documentation.
 Please help us [keep these instructions up-to-date][contribute]!
 
 
+## Linux
+
+For Ubuntu 20.04LTS, these commands install the basic build system:
+
+	sudo apt-get install python3 python3-pip scons git gcc-arm-none-eabi openocd \
+	  gcc build-essential libboost-all-dev
+	pip3 install modm gdbgui
+
+!!! warning "OpenOCD for recent targets"
+	The latest OpenOCD release v0.10.0 (as of May 2020) is too old for some targets
+	(STM32G0, STM32G4, STM32F7). To program these targets you need to compile the
+	[HEAD version of OpenOCD from source][openocd-source].
+
+!!! warning "Python 3 on Ubuntu ≤ 19.10"
+	Ubuntu ≤ 19.10 unfortunately defaults to Python 2, however, our SCons tools
+	require Python 3. For a less intrusive way to run all scons scripts with
+	Python 3 add this to your `.bashrc` or `.zshrc`:
+
+		alias scons="/usr/bin/env python3 $(which scons)"
+
+	If you get errors about missing `pyelftools`, check that you're really
+	running SCons with Python 3!
+
+!!! warning "arm-none-eabi-gcc on Ubuntu ≤ 19.10"
+	Ubuntu ≤ 19.10 does not provide a recent arm-none-eabi-gcc toolchain.
+	Install the ARM toochain by downloading [the pre-built version][gcc-arm-toolchain]
+	for 64-bit Linux and adding its `/bin` directory to your path.
```

Comment:
Wouldn't this conflict with the above install instructions? The user first does `sudo apt-get install gcc-arm-none-eabi`, then manually installs this toolchain and now has both toolchains in the path in probably undefined order?

Reply:
I would assume the user does not install the outdated GCC version when reading this warning.


> probably undefined order

Depends the path order in his `$PATH`.

Reply:
> Depends the path order in his $PATH.

I meant from our perspective undefined when they open an issue about it.

> I would assume the user does not install the outdated GCC version when reading this warning.

I would assume the user immediately executes first string that contains `sudo apt-get install`, mostly because that's what I would do ;-P

Reply:
`... and adding its /bin directory to the beginning of your $PATH.` should fix the issue and works in case the user already has installed any arm-none-eabi-gcc.